### ISSUE TITLE
README installation instructions do not work (Brackets Sprint 24)

### ISSUE THREAD

**Issue by jfsiii** [adobe-research/theseus#5]
TL;DR; Just use File -> Install Extension: https://s3.amazonaws.com/theseus-downloads/theseus-0.2.8.zip

More info at http://screencast.com/t/HtXNBgVI

Doesn't work:
- Following the instructions and putting `brackets-theseus` in the `extensions` directory.

Does work:
- Rename `brackets-theseus` to `thesus` and put it in the `extensions\user` directory.
- Using the built-in Install Extension feature (File -> Install Extension or "Lego+" icon) and pasting the zip file.

My preference would be to update the README to detail the Install Extension approach. It's much less work (and much more successful).
